README.md

```diff
@@ -1,58 +1,81 @@
 ```sudo ./run-health-checks.sh [~/health.log]```
-   - Default log file path is set to the current directory
+  - Default log file path is set to the current directory
 - Invoke health checks directly:
 ```sudo nhc -c ./conf/"CONFNAME".conf -l ~/health.log -t 300```
-   - To use a different log file location, specify the full path.
+  - To use a different log file location, specify the full path.
 - Adding more tests to the configuration files may require modifying the time flag (-t) to avoid timeout. For the default tests provided we recommend setting the timing to 300 seconds but this may vary from machine to machine.
 
-### _References_ ###
-- [LBNL Node Health Checks](https://github.com/mej/nhc)
-- [Azure HPC Images](https://github.com/Azure/azhpc-images)
+## Distributed NHC ##
 
-## Distributed NHC
 AzureHPC Node Health Checks also comes bundled with a distributed version of NHC, which is designed to run on a cluster of machines and report back to a central location. This is useful for running health checks on a large cluster with dozens or hundreds of nodes.
 
 See [Distributed NHC](./distributed-nhc/README.md) for more information.
 
-## Contributing
+## Health Checks ##
+
+Many of the hardware checks are part of the default NHC project. If you would like to learn more about these check out the [Node Health Checks project](https://github.com/mej/nhc).
+
+The following are Azure custom checks added to the existing NHC suite of tests:
+
+| Check | Component Tested | nd96asr_v4 expected| nd96amsr_a100_v4 expected | nd96isr_h100_v5 expected | hx176rs expected | hb176rs_v4 expected |
+|-----|-----|-----|-----|-----|-----|-----|
+| check_gpu_count | GPU count | 8 | 8 | 8 | NA | NA |
+| check_gpu_xid | GPU XID errors | not present | not present | not present | NA | NA |
+| check_nvsmi_healthmon | Nvidia-smi GPU health check | pass | pass | pass | NA | NA |
+| check_cuda_bw | GPU DtH/HtD bandwidth | 24 GB/s | 24 GB/s | 52 GB/s | NA | NA |
+| check_gpu_ecc | GPU Mem Errors (ECC) |  20000000 | 20000000 | 20000000 | NA | NA |
+| check_gpu_clock_throttling | GPU Throttle codes assertion | not present | not present | not present | NA | NA |
+| check_nccl_allreduce | GPU NVLink bandwidth | 228 GB/s | 228 GB/s | 460 GB/s | NA | NA |
+| check_ib_bw_loop_gdr | IB HCA device with GPU Direct RDMA(GDR) bandwidth | 180 GB/s | 180 GB/s | 380 GB/s | NA | NA |
+| check_nccl_allreduce_ib_loopback | GPU/GPU Direct RDMA(GDR) + IB device bandwidth | 18 GB/s | 18 GB/s | NA | NA | NA |
+| check_hw_topology | IB/GPU device topology/PCIE mapping | pass | pass | pass | NA | NA |
+| check_ib_link_flapping | IB link flap occurence | not present | not present | not present | not present | not present |
+| check_cpu_stream | CPU compute/memory bandwidth | NA | NA | NA | 665500 MB/s | 665500 MB/s |
+| check_ib_bw | IB device (non GDR) bandwidth | NA | NA | NA | 390 GB/s | 390 GB/s |
```

Comment:
Please split this into two:

For H-series, pls do non-GDR, and for NDv4/5 - pls do GDR.

Reply:
![image](https://github.com/Azure/azurehpc-health-checks/assets/70273488/47897fc1-0d4c-46fb-820d-ada738b683ea)
Is this what you were thinking (changing the component discription)? Or did you have in mind to rename the check's name (i.e. check_ib_bw_loop_gdr -> check_ib_bw_gdr and check_ib_bw-> check_ib_bw_non-gdr)?

Reply:
awesome, thanks. 

Yes, to both, we can keep the naming consistent.
- check_ib_bw
- check_ib_bw_gdr

Reply:
addressed